format-delimited/src/test/java/io/cdap/plugin/format/delimited/common/DataTypeDetectorUtilsTest.java

```diff
@@ -96,7 +96,7 @@ public void testDetectDataTypeOfEachDatasetColumn() {
   }
 
   @Test
-  public void testDetectDataTypeOfRowValue() {
+  public void testDetectDataTypeOfRowValue() throws RuntimeException {
```

Comment:
For runtime exception there is no need to add `throws`

Reply:
Removed